src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const property of action.keysToRemove) {
+          if (property in state) {
+            delete state[property];
+          }
+        }
+        break;
+
+      case 'clear':
+        for (const property in state) {
+          delete state[property];
+        }
+        break;
+
+      default:
```

Comment:
return error for default case to explain what happened

Reply:
I tried to return console.log('error') and I also tried to return alert('error') but the linter wouldn't let me do it. What else can I return in this case?

Reply:
I made return 'error'; is this option suitable?